Client/src/Components/Profile/UnitManagementSection/UnitRow.tsx

```diff
@@ -0,0 +1,49 @@
+import { EquationEvaluate, defaultErrorHandler } from "react-equation"
+import { Grid, IconButton } from "@material-ui/core"
+
+import ClearIcon from '@material-ui/icons/Clear'
+import { FastField } from "formik"
+import { MuiTextFieldFormik } from '../../Forms/FormikFields'
+import React from 'react'
+
+interface IProps {
+  index: number,
+  onRemoveUnitClick: (index: number) => void,
+  conversionFormula?: string,
```

Comment:
Dont we always need to specify a conversion formula?

Reply:
Yes. I believe I initially added that because new units didn't have a conversion formula set initially. Will check to see if I can remove the question mark.